src/pages/Stream/Views/Manage/Info.tsx

```diff
@@ -44,14 +46,13 @@ const InfoData = (props: { isLoading: boolean }) => {
 	const [info] = useStreamStore((store) => store.info);
 	const [currentStream] = useAppStore((store) => store.currentStream);
 
-	const createdAt = _.chain(info)
-		.get('created-at', '')
-		.thru((val) => (val !== '' ? dayjs(val).format('HH:mm A DD MMM YYYY') : '-'))
-		.value();
-	const firstEventAt = _.chain(info)
-		.get('first-event-at', '')
-		.thru((val) => (val !== '' ? dayjs(val).format('HH:mm A DD MMM YYYY') : '-'))
-		.value();
+	const createdAt = _.get(info, 'created-at', '-');
+	const firstEventAt = _.get(info, 'first-event-at', '-');
+	const createdAtParsed = getDateTimeWithTZ(createdAt);
+	const firstEventAtParsed = getDateTimeWithTZ(firstEventAt);
+	const validCreatedAt = createdAtParsed !== 'Invalid date' ? createdAtParsed : '-';
```

Comment:
This is not the right way to do it. 
I'd have to check and compare formatted date with 'Invalid date' everywhere I use `getDateTimeWithTZ`